lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -929,27 +931,28 @@ class MongoClientInterface {
                 // does not work because there cannot be null bytes
                 // in $regex
 
-                return async.parallel({
-                    bucketList: done => this.getBucketAttributes(
+                return async.waterfall([
+                    next => this.getBucketAttributes(
                     bucketName, log, (err, bucketInfo) => {
                         if (err) {
-                            return done(errors.InternalError);
+                            log.debug('error occured in countItems');
```

Comment:
Should log the error here, and use level error instead of debug, since this is going to be a real error that needs troubleshooting (goes for all returns that swallow errors)

Reply:
will change the log levels